### PR TITLE
chore: bump version

### PR DESCRIPTION
**Pull request by Decodetalkers**
Log:

### CODE REVIEW

**Comment by Decodetalkers** [comment]
https://github.com/linuxdeepin/developer-center/issues/4599
https://github.com/linuxdeepin/developer-center/issues/4718

**Review by BLumia — APPROVED** [review]

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+dde-clipboard (1:6.0.6) unstable; urgency=medium
+
+  * chore: move elideText logic to itemdata
+  * chore: add a commandline option
+
+ -- chenhongtao <chenhongtao@deepin.org>  Wed, 20 Aug 2023 11:43:05 +0800
```
Comment (BLumia) [inline debian/changelog:6]: 这个日期不对吧

**Review by BLumia — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 1 above)
Comment (Decodetalkers) [reply debian/changelog]: 嗯，错了

**Review by BLumia — APPROVED** [review]

**Pull request merged**